scripts/qa_crowbarsetup.sh

```diff
@@ -2401,7 +2409,7 @@ function prepare_proposals()
 function set_dashboard_alias()
```

Comment:
I actually think it doesn't belong to this PR. And if the new step is used together with batch, proposal should not be called anyway, thus no ``set_dashboard_alias``